kernel/src/process_standard.rs

```diff
@@ -1386,14 +1387,44 @@ impl<C: 'static + Chip> ProcessStandard<'_, C> {
                         "Process not enabled flash={:#010X}-{:#010X} process={:?}",
                         app_flash.as_ptr() as usize,
                         app_flash.as_ptr() as usize + app_flash.len() - 1,
-                        process_name
+                        process_name.unwrap_or("(no name)")
```

Comment:
I don't understand this change. Is this related?

Reply:
Its not related, but this would display the process name instead of Some(process_name). I suggest the same change at line 1452.